libretroshare/src/retroshare/rspeers.h

```diff
@@ -227,6 +238,7 @@ struct RsPeerDetails : RsSerializable
 	uint32_t trustLvl;
 	uint32_t validLvl;
 
+    bool skip_signature_validation;
```

Comment:
should this be renamed to `skip_pgp_signature_validation` for easier understanding?